__tests__/app.test.js

```diff
@@ -9,39 +9,73 @@ afterAll(() => db.end());
 
 beforeEach(() => seed(testData));
 
-describe('Error handling', () => {
-    describe("Status 404, path not found", () => {
-        test("passed an invalid path returns status 404 and a 'path not found' msg", () => {
-          return request(app)
-            .get("/api/banana")
-            .expect(404)
-            .then(({ body }) => {
-              expect(body.msg).toBe("path not found");
-            });
+describe("Error handling", () => {
+  describe("Status 404, path not found", () => {
+    test("passed an invalid path returns status 404 and a 'path not found' msg", () => {
+      return request(app)
+        .get("/api/banana")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("path not found");
         });
-      });
+    });
+  });
+});
 
-})
+describe("GET requests", () => {
+  describe("GET /api/categories", () => {
+    test("Status: 200, returns an array of catagory objects", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then(({ body }) => {
+          const { category } = body;
+          expect(category).toBeInstanceOf(Array);
```

Comment:
We do not need to check if it is an array since we are using a forEach, so we can delete this line 🙂